frontend/locales/en/common.ts

```diff
@@ -29,6 +29,24 @@ const common = {
     "repeat new password": "repeat new password",
     "new email": "new email",
     "repeat new email": "repeat new email",
+    "choose project name": "project name",
+    "project name placeholder": "project name",
+    "enter project info": "project info",
+    "project info placeholder": "project info",
+    "choose version control URL": "github",
+    "version control placeholder": "github link",
```

Comment:
I specifically chose to use the term VC so we wouldn't use "github". There are other VC. Think gitlab, ...